client/src/components/layout/DestinationShow.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react";
+
+const DestinationShow = (props) => {
+  const [destination, setDestination] = useState({
+    id: "",
+    name: "",
+    type: "",
+    location: "",
+    description: "",
+    website: "",
+  });
+  const { id, name, type, location, description, website } = destination;
+  const getDestination = async () => {
+    const destinationId = props.match.params.id;
+    try {
+      const response = await fetch(`/api/v1/destinations/${destinationId}`);
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`;
+        const error = new Error(errorMessage);
+        throw error;
+      }
+      const body = await response.json();
+      const destination = body.destination;
+      setDestination(destination);
+    } catch (err) {
+      console.error(`Error in fetch ${err.message}`);
+    }
+  };
+  useEffect(() => {
+    getDestination();
+  }, []);
+  return (
```

Comment:
add a new line between functions - after declaring state, before and after `getDestination`, and a space between the useEffect and return so that your code doesn't run together and instead has some separation to help with readability